docs/api.md

```diff
@@ -0,0 +1,30 @@
+# OSM Teams
```

Comment:
the output of swagger-markdown is not super useful. I wish it included the schemas.

Reply:
haven't found a way to do this yet but can look into it more in the future